connect-example/src/main/java/com/stripe/android/connect/example/data/SettingsService.kt

```diff
@@ -42,7 +42,7 @@ class SettingsService @Inject constructor(@ApplicationContext context: Context)
     fun getAppearanceIdFlow(): Flow<AppearanceInfo.AppearanceId?> {
         return observePref(APPEARANCE_ID_KEY) { prefs, key ->
             val appearanceId = prefs.getString(key, null) ?: return@observePref null
-            AppearanceInfo.AppearanceId.valueOf(appearanceId)
+            AppearanceInfo.AppearanceId.entries.firstOrNull { it.name == appearanceId }
```

Comment:
👀 this fixes a crash if any appearance id's change and can't be found anymore (something I ran into myself)